climada/util/coordinates.py

```diff
@@ -290,22 +290,26 @@ def dist_approx(lat1, lon1, lat2, lon2, log=False, normalize=True,
         If False, assume that all longitudinal values lie within a single interval of size 360
         (e.g., between -180 and 180, or between 0 and 360) and such that the shortest path between
         any two points does not cross the antimeridian according to that parametrization. If True,
-        a suitable interval is determined using `lon_bounds` and the longitudinal values are
-        reparametrized accordingly using `lon_normalize`. Note that this option has no effect when
-        using the "geosphere" method because it is independent from the parametrization.
-        Default: True
+        a suitable interval is determined using :py:func:`lon_bounds` and the longitudinal values
+        are reparametrized accordingly using :py:func:`lon_normalize`. Note that this option has
+        no effect when using the "geosphere" method because it is independent from the
+        parametrization. Default: True
     method : str, optional
         Specify an approximation method to use:
-            * "equirect": Distance according to sinusoidal projection. Fast, but inaccurate for
-              large distances and high latitudes.
-            * "geosphere": Exact spherical distance. Much more accurate at all distances, but slow.
+
+        * "equirect": Distance according to sinusoidal projection. Fast, but inaccurate for
```

Comment:
This is unrelated to the main issue in this PR. Correcting indentation and adding empty lines before and after bullet lists fixes a few sphinx warnings.